src/components/menu-top/LoadShow.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        Load Show
+      </p>
+    </header>
+
+    <section class="modal-card-body">
+      <b-field class="file">
+        <b-upload
+          v-model="file"
+          accept=".shw"
+          @input="loadShow"
+        >
+          <a class="button is-primary">
+            <b-icon icon="upload" />
+            <span>Click to load</span>
+          </a>
+        </b-upload>
+        <span
+          v-if="file"
+          class="file-name"
+        >
+          {{ file.name }}
+        </span>
+      </b-field>
+      <span
+        v-if="parseError"
+      >
+        <b-message
+          title="Error"
+          type="is-danger"
+          aria-close-label="Close message"
+        >
+          {{ parseError }}
+        </b-message>
+      </span>
+      <span
+        v-if="show"
+      >
+        <b-field>
+          Marchers: {{ numMarchers }}
```

Comment:
Similar suggestion for sheets
```suggestion
          Number of marchers: {{ numMarchers }}
```

Reply:
Done